### DISCUSSION TITLE
Standup (Monday, May 9, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#47] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Helped to draw a class diagram

**What are your priorities for the next 24 hours?**:
 - Plan backend development for this week

**Blockers**:
 - We do not have a plan yet

**Shoutouts**:
 - @CarlosCRG19 and @DavBot02 for finishing our first sprint

**Reply by CarlosCRG19** [reply]
It has not finished yet, though, heheh

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Component diagram
 - Figma wireframes

**What are your priorities for the next 24 hours?**:
 - Setup environment variables
 - Meetup with the frontend team

**Blockers**:
 - None

**Shoutouts**:
 - @everyone for their hard work on the deliverable

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished class diagram with @antonieto 

**What are your priorities for the next 24 hours?**:
 -  Define backend priorities with @antonieto and start working on them
 - Checking on @CarlosCRG19  notes about our code and fix it

**Blockers**:
 - Lack of time due some other classes 

**Shoutouts**:
 - @CarlosCRG19  for reviewing our code

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished first functional "create job offer" screen

**What are your priorities for the next 24 hours?**:
 - Clean code with @CarlosCRG19 

**Blockers**:
 -

**Shoutouts**:
 - @CarlosCRG19 for finishing the diagrams

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing much

**What are your priorities for the next 24 hours?**:
 - Do something (?

**Blockers**:
 - 

**Shoutouts**:
 - @CarlosCRG19 fo coordinating everything
